CONTRIBUTING.md

```diff
@@ -4,23 +4,43 @@
 
 We use [pnpm](https://pnpm.io) to manage our dependencies.  
 You can [install it](https://pnpm.io/installation) using your preferred
-method, but we recommend using [Corepack](https://nodejs.org/api/corepack.html#supported-package-managers) so that you
-keep your package manager version in sync with ours.
+method, but we recommend using [Corepack](https://nodejs.org/api/corepack.html#supported-package-managers) so that you keep your package manager version in sync with ours.
 
 Make sure you run **Node 20** or the dependencies' installation will fail.  
 We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your Node version.
 
 Finally, run `pnpm install`.
 
+## Running
+
+### Minimal setup
+
+Copy [`.env.example`](./.env.example) to `.env` and fill you discord token.
```

Comment:
when i git clone, .env.example already exist. 
Just need to rename .env.example to .env and fill the discord token.